opencensus/trace/BUILD

```diff
@@ -135,6 +136,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "traceparent",
```

Comment:
I would call this trace context to be consistent with all the other languages

Reply:
Done. Should I also rename the functions from TraceParent? (I find it a bit weird that the header doesn't match the other names but consistency is important)